tests/browser/features/happy.feature

```diff
@@ -49,6 +49,8 @@ Feature: Happy path
     Then they should see the pensions-benefits-short-tax-return question page
     When they enter correct pension details
     Then they should see the enter-recent-self-assessment-payment question page
+    When they enter wrong self assessment payment details
```

Comment:
We should create a new Validation feature test, this is just for happy path scenarios